spec/models/registration_spec.rb

```diff
@@ -22,4 +22,17 @@
       end
     end
   end
+  describe 'after destroy' do
+    it 'destroys purchased tickets' do
+      conference = build(:conference)
+      registration1 = build(:registration, conference: conference)
+      registration1.save
+      ticket_purchase = build(:ticket_purchase, conference: conference)
+      ticket_purchase.save
```

Comment:
then should i modify lines 14-18 of this file too because they had the same flow?